src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -1242,11 +1242,23 @@ private void PerformSyncOperation<TOperation>(ref OperationQueue<TOperation> que
             }
         }
 
-        public SocketError Accept(byte[] socketAddress, ref int socketAddressLen, int timeout, out IntPtr acceptedFd)
+        private bool ShouldRetrySyncOperation(out SocketError errorCode)
+        {
+            if (_nonBlockingSet)
+            {
+                errorCode = SocketError.Success;
```

Comment:
Why Success?  Or is it that this will just be entirely ignored (since we're returning true) but you need some value to satisfy the out?  If the latter, consider adding a small comment to that effect.

Reply:
It will be ignored, just need an out value.